Orange/tests/test_domain.py

```diff
@@ -368,7 +368,7 @@ def test_conversion(self):
         x, y, metas = domain.convert([42, 13, "White"])
         assert_array_equal(x, np.array([42, 13]))
         assert_array_equal(y, np.array([0]))
-        assert_array_equal(metas, np.array([Unknown, Unknown, None]))
```

Comment:
Not exactly sure what's the deal here. The previous implementation seems to have returned `None` for unknown `StringVariable`s?